commands.js

```diff
@@ -2081,15 +2081,15 @@ exports.commands = {
 				if (!stdout) {
 					connection.popup("The modlog is empty. (Weird.)");
 				} else {
-					connection.popup("|wide||html|<p>The last " + lines + " lines of the Moderator Log of " + roomNames + ":</p>" + stdout);
+					connection.popup("|wide||html|<p>The last " + lines + " lines of the Moderator Log of " + roomNames + ".</p><p><small>[" + Tools.toTimeStamp(new Date(), {hour12: true}) + "]</small> Current server time.</p>" + stdout);
```

Comment:
Done. http://imgur.com/EQLpTWU

Also made the <small> tag cover the whole line, which makes it seem more like it's part of the header and less like a modlog entry.